### PR TITLE
CIRC-6408 Added Graphite-style queries with autocomplete

### PR DESCRIPTION
**Pull request by graefen**

### CODE REVIEW

**Comment by graefen** [comment]
This PR covers CIRC-6408 & CIRC-6409

File: src/config_ctrl.ts [hunk 1]
```diff
@@ -6,7 +6,8 @@ export class IrondbConfigCtrl {
 
   /** @ngInject */
   constructor($scope) {
-    this.current.jsonData.irondbType = this.current.jsonData.irondbType || 'standalone';
+    this.current.jsonData.irondbType = this.current.jsonData.irondbType || 'hosted';
```
Comment (bdunavant) [inline src/config_ctrl.ts:9]: Was this change to the default intended?

**Review by bdunavant — APPROVED** [review]
LGTM

File: src/config_ctrl.ts (hunk 1 above)
Comment (graefen) [reply src/config_ctrl.ts:9]: yes, but I don't remember if I had asked...should 'standalone' be the default instead?

**Pull request merged**